### PR TITLE
Fix/testing bug fixes

### PR DESCRIPTION
**Pull request by shreeyash07**
## Changes
- Add missing feature in home page

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: src/components/domain/ActiveOperationMap/index.tsx [hunk 1]
```diff
@@ -259,6 +290,40 @@ function ActiveOperationMap(props: Props) {
             className={_cs(styles.activeOperationMap, className)}
             heading={heading}
             withHeaderBorder
+            headerDescriptionContainerClassName={styles.filters}
+            headerDescription={(
```
Comment (frozenhelium) [inline src/components/domain/ActiveOperationMap/index.tsx]: Let's use `filters` prop for Filters

File: src/components/domain/AppealsTable/index.tsx [hunk 2]
```diff
@@ -188,6 +203,18 @@ function AppealsTable(props: Props) {
...
+                        label={strings.appealsTableStartDate}
+                        onChange={setFilterField}
+                        value={filter.startDate}
+                    />
+                    <DateInput
+                        name="endDate"
+                        label={strings.appealsTableEndDate}
+                        onChange={setFilterField}
+                        value={filter.endDate}
+                    />
```
Comment (frozenhelium) [inline src/components/domain/AppealsTable/index.tsx:217]: Let's move these and other filters to `filters` prop

File: src/components/domain/HighlightedOperations/OperationCard/index.tsx [hunk 3]
```diff
@@ -40,10 +42,11 @@ function OperationCard(props: Props) {
             appeals,
             countries = [],
         },
-        subscriptionMap,
+        isSubscribed = false,
```
Comment (frozenhelium) [inline src/components/domain/HighlightedOperations/OperationCard/index.tsx:45]: This property is already in `data`

File: src/components/domain/HighlightedOperations/OperationCard/index.tsx [hunk 4]
```diff
@@ -103,7 +104,14 @@ function OperationCard(props: Props) {
...
             className={_cs(styles.operationCard, className)}
-            heading={name}
+            heading={(
+                <Link
+                    to="emergenciesLayout"
+                    urlParams={{ emergencyId: id }}
+                >
+                    {name}
+                </Link>
+            )}
```
Comment (frozenhelium) [inline src/components/domain/HighlightedOperations/OperationCard/index.tsx]: Let's see if we can support links in heading from Container itself

File: src/components/domain/ActiveOperationMap/index.tsx [hunk 5]
```diff
@@ -106,24 +116,44 @@ function ActiveOperationMap(props: Props) {
...
+        limit,
+    } = useFilterState<{
+        appeal?: AppealTypeOption['key'],
+        displacement?: number,
+        startDate?: string,
+        endDate?: string,
+    }>(
+        {},
+        undefined,
+        200,
```
Comment (tnagorra) [inline src/components/domain/ActiveOperationMap/index.tsx]: 9999

**Review by tnagorra — APPROVED** [review]

File: src/components/domain/HighlightedOperations/OperationCard/index.tsx [hunk 6]
```diff
@@ -40,10 +42,11 @@ function OperationCard(props: Props) {
             appeals,
             countries = [],
         },
-        subscriptionMap,
+        isSubscribed = false,
     } = props;
 
     const { invalidate } = useContext(DomainContext);
+    const { userAuth: userDetails } = useContext(UserContext);
```
Comment (frozenhelium) [inline src/components/domain/HighlightedOperations/OperationCard/index.tsx]: Let add a hook for userAuth

File: src/views/Home/index.tsx [hunk 7]
```diff
@@ -19,6 +22,13 @@ import { useRequest } from '#utils/restRequest';
 import i18n from './i18n.json';
 import styles from './styles.module.css';
 
+function percent(value: number, total: number) {
+    if (isNotDefined(value) || isNotDefined(total)) {
+        return 0;
+    }
+    return (value / total) * 100;
+}
```
Comment (frozenhelium) [inline src/views/Home/index.tsx]: Let's also check for `total === 0`

File: src/views/Home/index.tsx [hunk 8]
```diff
@@ -48,13 +58,35 @@ export function Component() {
...
-                                description={strings.homeKeyFiguresActiveDrefs}
+                                description={(
+                                    <div className={styles.description}>
+                                        {strings.homeKeyFiguresActiveDrefs}
+                                        <div className={styles.descriptionIcon}>
+                                            <AlertInformationLineIcon />
+                                            <Tooltip
+                                                description={strings.keyFiguresDrefDescription}
+                                            />
+                                        </div>
```
Comment (frozenhelium) [inline src/views/Home/index.tsx]: Let's implement this feature in `KeyFigure`

File: src/views/Home/styles.module.css [hunk 9]
```diff
@@ -8,6 +8,16 @@
...
             background-color: var(--go-ui-color-white);
             height: 100%;
+
+            .description {
+                display: flex;
+
+                .description-icon {
+                        display: flex;
+                        align-items: center;
+                        font-size: var(--go-ui-font-size-2xl);
```
Comment (frozenhelium) [inline src/views/Home/styles.module.css]: should use `var(--go-ui-font-size-icon-multiplier)`

**Review by frozenhelium — CHANGES_REQUESTED** [review]

**Pull request merged**